smithy-aws-protocol-tests/model/restJson1/http-query.smithy

```diff
@@ -105,9 +105,9 @@ apply AllQueryStringTypes @httpRequestTests([
             "QueryParamsStringKeyB=Bar",
         ],
         params: {
-            queryParamsMapOfStrings: {
-                "QueryParamsStringKeyA": "Foo",
-                "QueryParamsStringKeyB": "Bar",
+            queryParamsMapOfStringList: {
+                "QueryParamsStringKeyA": ["Foo"],
+                "QueryParamsStringKeyB": ["Bar"],
```

Comment:
I agree that we should have this test, but why do we need to get rid of the old one?

Reply:
If `@httpQueryParams` is on a member, it has to be on a map of lists if any other `@httpQuery` member can have multiple values. So this isn't really about removing or adding tests, it's about making this test compatible with servers.